types/network/network.go

```diff
@@ -21,6 +22,13 @@ type IPAMConfig struct {
 	AuxAddress map[string]string `json:"AuxiliaryAddresses,omitempty"`
 }
 
+// IPAMInfo represents IPAM operational information
+type IPAMInfo struct {
+	PoolID  string `json:",omitempty"`
```

Comment:
@aboch ,
Many thanks for comment.

As you suggested here https://github.com/docker/docker/issues/23025, for handling identical cidr pools, address-spaces can be used but many networking backends still have problems in supporting address-spaces. In that case, these drivers can invoke "network inspect" rest api to get PoolID and pick the correct pool.

Also showing PoolID will provide more visiblity to user, in case user is aware of IPAM pools.

In case your concern is just not to use "Info" word for PoolID as it is not an operational info, Will it work for you if struct is renamed to something this "IpamData"?

Looking forward for your valuable inputs.